### DISCUSSION TITLE
Sample file

### DISCUSSION THREAD

**Discussion by doan-van** [MarkusPic/intensity_duration_frequency_analysis#8] [General]
Hi! Can I have sample file ehyd_112086.parquet to run the program?

**Comment by MarkusPic** [comment]
I created this file with this script: <https://github.com/MarkusPic/ehyd_tools/blob/main/example/example_python_api.ipynb>

You can download rain time series for Austrian measurement stations with this script.
For example, the file you want is for Graz.

**Reply by doan-van** [reply]
Let me try it. Appreciate your answer!
